lib/backend/model/bgpconfig.go

```diff
@@ -17,13 +17,18 @@ package model
 import (
 	"fmt"
 	"reflect"
+	"regexp"
+
+	log "github.com/Sirupsen/logrus"
 
 	"github.com/projectcalico/libcalico-go/lib/errors"
 )
 
 var (
-	typeGlobalBGPConfig = rawStringType
-	typeHostBGPConfig   = rawStringType
+	matchGlobalBGPConfig = regexp.MustCompile("^/?calico/bgp/v1/global/(.+)$")
```

Comment:
We didn't previously support list operations for config.  I was adding that in, although for the life of me I can't remember why.